findingcheapestgpus.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: 'Finding the Cheapest GPUs'
```

Comment:
Something that we had discussed before for sales pitches was using something like "Most affordable" instead of "cheapest" because "cheapest" could have negative connotations. Do you think this is worth distinguishing for docs or am I splitting hairs?

Reply:
I think that if you've seen that, then its best to put it in, thanks. Will change